### ISSUE TITLE
Add official Ubuntu PPA

### ISSUE THREAD

**Issue by Timidger** [way-cooler/way-cooler#470]
[There](https://github.com/way-cooler/way-cooler/issues/469) are [a](https://github.com/way-cooler/way-cooler/issues/395) [lot](https://github.com/way-cooler/way-cooler/issues/327) of problems [with](https://github.com/way-cooler/way-cooler/issues/326) installing Way Cooler [on Ubuntu](https://github.com/way-cooler/way-cooler/issues/360).

To remedy this, there should eventually be an officially hosted Ubuntu PPA where we can point users to.

**Comment by Oddly** [comment]
How about adding a [snap](https://snapcraft.io/)?

**Comment by Timidger** [comment]
That would also be acceptable. I'm not looking to do that any time soon, but if someone gets to it before me I'll add it to the README.

**Comment by Sunderland93** [comment]
I can create PPA for Way-Cooler and support it, but it may be better to wait wlroots-version?

**Comment by Timidger** [comment]
Yes it would be better to wait until we switch to wlroots. In fact best to wait until 1.0 entirely, which won't be for a while. I plan to start dealing with packaging once I'm closer to 1.0, so I'll reach out to you then.

**Comment by tmerse** [comment]
Any estimate how far we are still away from this?

**Comment by Timidger** [comment]
It's not even usable (except for older versions, such as 0.8) so it's way down on the bucket list.
